src/utils/toast.tsx

```diff
@@ -110,12 +110,21 @@ export const createTalentLayerIdTransactionToast = async (
 
     return entityId;
   } catch (error) {
-    const parsedEthersError = getParsedEthersError(error as EthersError);
+    const errorMessage = getParsedErrorMessage(error);
     console.error(error);
     toast.update(toastId, {
       type: toast.TYPE.ERROR,
-      render: `${messages.error}: ${parsedEthersError.errorCode} - ${parsedEthersError.context}`,
+      render: errorMessage,
     });
   }
   return;
 };
+
+function getParsedErrorMessage(error: any) {
+  const parsedEthersError = getParsedEthersError(error as EthersError);
+  if (parsedEthersError.errorCode === RETURN_VALUE_ERROR_CODES.REJECTED_TRANSACTION) {
```

Comment:
I added a custom case for this error code, otherwise the error message nearly took the whole page.